library/src/webapp/js/lang-datepicker/lang-datepicker.js

```diff
@@ -1110,6 +1132,8 @@ isRTL: false,
 showMonthAfterYear: false,
 yearSuffix: ''};
 $.datepicker.regional['ru'] = {
+openDatepicker: 'Открыть календарь',
+datepickerOpened: 'Календарь открыт',
```

Comment:
How did you do these translations? Are they from somewhere else?

Reply:
Google Bard, I have added the translation for top 5 used languages with the help of AI.